engine/language-server/src/main/scala/org/enso/languageserver/search/SuggestionsHandler.scala

```diff
@@ -264,54 +262,6 @@ final class SuggestionsHandler(
         )
       )
 
-    case msg: Api.ExpressionUpdates if state.isSuggestionUpdatesRunning =>
-      state.suggestionUpdatesQueue.enqueue(msg)
-
-    case Api.ExpressionUpdates(_, updates) =>
```

Comment:
Why this change? Seems rather unrelated. Or is that feature no longer used?

Reply:
Yes, after the component browser is finished, and GUI can handle suggestions itself, we don't need to do this logic anymore